unsafe/src/test/java/org/apache/spark/unsafe/types/UTF8StringSuite.java

```diff
@@ -148,26 +150,26 @@ public void concatWsTest() {
       EMPTY_UTF8,
       concatWs(sep, EMPTY_UTF8));
     assertEquals(
-      fromString("ab"),
-      concatWs(sep, fromString("ab")));
+            fromString("ab"),
```

Comment:
Why this? Could you update the code style of Java in your IDE? using indent as 2 space.